client/jetpack-onboarding/steps/business-address.jsx

```diff
@@ -71,19 +74,20 @@ class JetpackOnboardingBusinessAddressStep extends React.PureComponent {
 				<FormattedHeader headerText={ headerText } subHeaderText={ subHeaderText } />
 
 				<Card className="steps__form">
-					<form>
+					<form onSubmit={ this.handleSubmit }>
 						{ map( this.fields, ( fieldLabel, fieldName ) => (
 							<FormFieldset key={ fieldName }>
 								<FormLabel htmlFor={ fieldName }>{ fieldLabel }</FormLabel>
 								<FormTextInput
+									autoFocus={ fieldName === 'name' }
 									id={ fieldName }
 									onChange={ this.getChangeHandler( fieldName ) }
+									required
```

Comment:
This will make `state` field also required, and it wasn't required in the original plugin: https://github.com/Automattic/jetpack-onboarding/blob/master/client/components/steps/business-address.jsx#L70